test/Controllers/ServicesTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/*
+ * Services SDK Controller Test Case
+ */
+
+namespace GonebusyTest\Controllers;
+
+use PHPUnit\Framework\TestCase;
+
+use GonebusyLib\Configuration;
+use GonebusyLib\GonebusyClient;
+use GonebusyLib\Models\CreateServiceBody;
+use GonebusyLib\Models\UpdateServiceByIdBody;
+
+class ServicesTest extends TestCase
+{
+    /**
+     * To contain the GonebusyLib\GonebusyClient
+     */
+    protected $client;
+
+    /**
+     * To contain the GonebusyLib\Controllers\ServicesController
+     */
+    protected $services;
+
+    /**
+     * Create the GoneBusy SDK client for each test.
+     */
+    public function setUp() {
+        $this->client = new GonebusyClient();
+        $this->services = $this->client->getServices();
+    }
+
+
+    /**
+     * Returns arbitrary service data.
+     * @param  string $type Should be 'create' or 'update'.
+     * @return  CreateServiceBody or UpdateServiceByIdBody object with unique data to send to API
+     */
+    private function serviceBody($type) {
+        $rand = rand();
+        switch($type) {
+            case 'create':
+                return new CreateServiceBody(
+                    "description", // REQUIRED
+                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
+                    "name", // REQUIRED
+                    NULL, // categories
+                    NULL, // price_model_id XXX defaults to inexistent id
```

Comment:
No, sorry I just meant that the API is behaving unexpectedly. I'm not trying to send a PriceModel ID, since it's optional I just send NULL (saving us lines of cose and yet another call to the API to create or find an actual PricingModel). But I noticed when you do send NULL (or nothing in REST), the resulting Service ends up with a random `price_model_id`:

![image](https://cloud.githubusercontent.com/assets/1477535/23937129/49d8077c-092c-11e7-9e90-fd24931dc31c.png)

which doesn't even exist:

![image](https://cloud.githubusercontent.com/assets/1477535/23937140/5f2340ce-092c-11e7-9a39-f33e014746ed.png)